driver/conftest.py

```diff
@@ -3,6 +3,8 @@
 
 from _pytest.config.argparsing import Parser
 
+BUNDLE_URL = "https://raw.githubusercontent.com/canonical/bundle-kubeflow/refs/heads/main/releases/latest/edge/bundle.yaml"
```

Comment:
@orfeas-k We could keep this as is, and when we branch off with track/1.9 we can update this to point to 1.9/stable